packages/utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@trezor/utils",
-    "version": "1.0.0",
+    "version": "1.0.1",
     "author": "Trezor <info@trezor.io>",
     "homepage": "https://github.com/trezor/trezor-suite/packages/utils",
```

Comment:
wrong link, it should be probably https://github.com/trezor/trezor-suite/tree/develop/packages/utils instead

Reply:
changed